### PR TITLE
docs: sync docs with foundation-ui PA-823

### PR DESCRIPTION
**Pull request by matteematt**
Thank you for contributing to the documentation.

Your Jira ticket is:
PA-823 has introduces some foundation-layout changes so I took the opportunity to sync the UI docs

Have you provided changes for all the relevant versions: next, 2022.4, 2023.1, etc?

Yes - only next changes

Have you checked all new or changed links?

Yes

Is there anything else you would like us to know?

No

For reference: 

- if you are an internal contributor:
  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 
- If you are an external contributor:
  - We have an [external contribution guide](../Type-of-contribution)

**This week's exciting excerpt from the style guide**
LISTS. Only use numbered lists when the sequence is important. Usually, that is a set of instructions.
For example:
1. Change the dictionary files.
2. Run genesisInstall.
3. Run remap.
In all other cases, use an **unnumbered** list. OK?

### CODE REVIEW

File: docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md [hunk 1]
```diff
@@ -26,18 +26,15 @@ You can customise:
 - the icon shown on the navigation bar and flyout menu (this shows the Genesis logo by default)
 - navigation links at the left-hand side of the navigation bar
 - the control buttons on the right-hand side of the navigation bar; these can be shown or hidden, and their behaviour controlled via event listeners
-- the contents of the flyout menu
+- The contents of the flyout menu
```
Comment (MrBrunoWolff) [inline docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md:29]: question:

should we capitalize the other items?

File: plugins/api-docs/index.js [hunk 2]
```diff
@@ -100,6 +100,7 @@ async function copyApiDocs(manifest, processedMap) {
     console.log("[api-docs-plugin] No packages awaiting processing.");
     return;
   }
+	console.log("[api-docs-plugin] Processing packages:", packagesToProcess)
```
Comment (MrBrunoWolff) [inline plugins/api-docs/index.js:103]: question: is this needed? 

Indentation seems a bit different from the other lines 👀

File: package.json [hunk 3]
```diff
@@ -73,7 +73,7 @@
     ]
   },
   "devDependencies": {
-    "@genesislcap/rollup-builder": "14.30.1", 
+    "@genesislcap/rollup-builder": "14.30.1",
```
Comment (MrBrunoWolff) [inline package.json:76]: question:

do we have `14.62.1` for this package?

**Review by MrBrunoWolff — COMMENTED** [review]
Adding some comments!

Happy with changes.

File: plugins/api-docs/index.js (hunk 2 above)
Comment (matteematt) [reply plugins/api-docs/index.js]: yes this is left over from debugging - good spot thanks

File: docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md (hunk 1 above)
Comment (matteematt) [reply docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md:29]: I have just synced the docs from the UI repo which already has reviews. I am personally going to leave this as it is but if the docs team want to change this then happy with that too (and then will resync with UI repo)

File: package.json (hunk 3 above)
Comment (matteematt) [reply package.json]: Yes, updated

File: docs/04_web/05_micro-front-ends/05_foundation-login/05_foundation-login.md [hunk 4]
```diff
@@ -40,12 +40,12 @@ An example of the change password screen: <br/>
...
 
 :::info
-If you build a project upon one of the Genesis seed apps, then the `Login` micro front-end will already be set-up for you.
+If you build a project upon one of the Genesis seed apps, then the `Login` micro front-end will already be set up for you.
 :::
 
 To enable this micro front-end in your application, follow the steps below.
 
-- Add `@genesislcap/foundation-login` as a dependency in your **package.json** file. Whenever you change the dependencies of your project, always you run the `$ npm run bootstrap` or `$ npm run bootstrap:ignore-scripts` command again. You can see more information on the [package.json basics](../../../../web/micro-front-ends/foundation-login/) page.
+1. Add `@genesislcap/foundation-login` as a dependency in your **package.json** file. Whenever you change the dependencies of your project, always run the `$ npm run bootstrap` or `$ npm run bootstrap:ignore-scripts` command again. You can see more information on the [package.json basics](../../../../web/micro-front-ends/foundation-login/) page.
```
Comment (matteematt) [inline docs/04_web/05_micro-front-ends/05_foundation-login/05_foundation-login.md:48]: Does it make sense to have a numerical list here when there is only one task?

File: docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md (hunk 1 above)
Comment (wjhendry) [reply docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md:29]: Check the entry on Lists in the style guide
https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b#b305f37d9b1d44e1bafeeabd06894b12

**Review by wjhendry — APPROVED** [review]

**Comment by wjhendry** [comment]
Just a note on micro front-end
This was the form I settled on after looking at the grammar: micro is an adjective describing the compound noun front-end (the hyphen makes the two words inseparable).
There is a case for micro-front-end. But I feel that that looks more complex.

File: docs/04_web/05_micro-front-ends/04_foundation-entity-management/04_foundation-entity-management.md [hunk 5]
```diff
@@ -21,25 +21,27 @@ tags:
...
-Two core components are used to manage the entities - the grid and the form. The grid contains an entity on each row and data in each column.
+Two core components are used to manage the entities - the grid and the form. 
+
+The grid contains an entity on each row and data in each column. Here is an example showing a grid used to manage counterparties:
 
-Example grid used to manage counterparties
 ![Example entity management grid](./docs/img/foundation-entity-management-grid.png)
 
-The other component is the form, and this is used to update an existing entity or create a new one. An example form is shown here.
+The other component is the form. This is used to update an existing entity or to create a new one. Here is an example :
```
Comment (matteematt) [inline docs/04_web/05_micro-front-ends/04_foundation-entity-management/04_foundation-entity-management.md:36]: Here is an example:

File: docs/04_web/05_micro-front-ends/04_foundation-entity-management/04_foundation-entity-management.md [hunk 6]
```diff
@@ -99,11 +102,17 @@ The functionality of the Entity Manager is customised through the properties you
 
 #### Resource
 
-`resourceName` refers to the name of either a `DATASERVER` query, a `ReqRep` definition or a specific `EVENT_HANDLER`. This essentially is the resource to be managed. In the [above example](#set-up), we use `ALL_COUNTERPARTYS` which is a `SDATASERVER` query, as, by convention, they all start with `ALL_`.
+`resourceName` refers to the name of a resource on the server. This could be:
+
+- a `query` in a Data Server
+- a `ReqRep` definition in a Request Server
+- an `eventHandler`in the Event Handler.
```
Comment (matteematt) [inline docs/04_web/05_micro-front-ends/04_foundation-entity-management/04_foundation-entity-management.md:109]: Remove the comma?

**Review by matteematt — COMMENTED** [review]
two small queries

**Pull request merged**